procrastinate/contrib/django/admin.py

```diff
@@ -26,11 +77,41 @@ def has_add_permission(self, request, obj=None):
     def has_delete_permission(self, request, obj=None):
         return False
 
+    @admin.display(description="Status")
+    def pretty_status(self, instance: models.ProcrastinateJob) -> str:
+        emoji = JOB_STATUS_EMOJI_MAPPING.get(instance.status, "")
+        return f"{emoji} {instance.status.title()}"
+
+    @admin.display(description="Task Name")
+    def short_task_name(self, instance: models.ProcrastinateJob) -> str:
+        *modules, name = instance.task_name.split(".")
+        return format_html(
+            "<span title='{task_name}'>{name}</span>",
+            task_name=instance.task_name,
+            name=".".join(m[0] for m in modules) + f".{name}",
+        )
+
+    @admin.display(description="Args")
+    def pretty_args(self, instance: models.ProcrastinateJob) -> str:
+        indent = 2 if len(instance.args) > 1 or len(str(instance.args)) > 30 else None
+        pretty_json = json.dumps(instance.args, indent=indent)
+        if len(pretty_json) > 2000:
+            pretty_json = pretty_json[:2000] + "..."
+        return format_html(
+            '<pre style="margin: 0">{pretty_json}</pre>', pretty_json=pretty_json
+        )
 
-admin.site.register(
-    [
-        models.ProcrastinateJob,
-        models.ProcrastinateEvent,
-    ],
-    ProcrastinateAdmin,
-)
+    @admin.display(description="Summary")
+    def summary(self, instance: models.ProcrastinateJob) -> str:
+        if last_event := instance.procrastinateevent_set.latest():  # type: ignore[attr-defined]
+            return mark_safe(
```

Comment:
Here, `mark_safe` is safe because external values got escaped when the template was rendered, so the result of the temple is actually safe